template/darwin/module/configuration.nix

```diff
@@ -13,5 +14,7 @@
       warn-dirty = false;
     };
   };
+
   programs.zsh.enable = true;
+  system.stateVersion = 6;
```

Comment:
What is this line required for? Seems somewhat random to have a stateVersion

Reply:
This is `nix-darwin` specific.